src/Google/Ads/GoogleAds/Util/V1/ResourceNames.php

```diff
@@ -283,7 +283,7 @@ public static function forAdGroupExtensionSetting($customerId, $adGroupId, $exte
     {
         return AdGroupExtensionSettingServiceClient::adGroupExtensionSettingName(
             $customerId,
-            "{$adGroupId}~{$extensionType}"
+            sprintf("%s~%s", $adGroupId, ExtensionType::name($extensionType))
```

Comment:
ExtensionType class has not been imported as in V3

Reply:
Sorry I requested re-review too early.